storage/tokudb/CMakeLists.txt

```diff
@@ -16,6 +16,14 @@ IF(NOT TOKUDB_OK)
   RETURN()
 ENDIF()
 
+set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/PerconaFT/cmake_modules")
+include(TokuFeatureDetection)
+
```

Comment:
I don't quite like that you pull in TokuFeatureDetection from PerconaFT here. And that it performs lots of checks even of tokudb is disabled.

Instead, I'd extend the TOKUDB_OK check to test for F_NOCACHE and O_DIRECT.

Reply:
Oh ok yes good point.

Reply:
Extending TOKUDB_OK is wrong because these 2 flags are optional features. Our machines don't have F_NOCACHE but we use TokuDB. Now with 10.2.11, MariaDB does not build TokuDB anymore.

Reply:
Hmm... I should have looked at the bug tracker first. This is already marked as fixed (but I don't know in which branch).